README.md

```diff
@@ -154,6 +154,14 @@ you should also include the following `script-src` directive:
 script-src https://sdk.twilio.com
 ```
 
+If you are enabling [Krisp Noise Cancellation](https://www.twilio.com/docs/video/noise-cancellation) for
+your local audio, and you are using the following `default-src self` directive, you should also add the
+`unsafe-eval` directive:
+
+```
+default-src self unsafe-eval
+```
```

Comment:
```
default-src 'self' 'unsafe-eval';
```

Reply:
What you've written is the client-side meta tag syntax. What I've specified is the server side syntax.